home/static/comp-tool/kb/disco/media.csv

```diff
@@ -1,78 +1,78 @@
-URI,SmallURI,LargeURI,ImageWidth,Type,Priority,Caption,Taxon,Character,State,UseFor,TipStyle
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-,,,,,,,,,,,
-resources/images/MCDBottom.jpg,,,100%,image-local,2,Bottom view of a Milk Chocolate Digestive ,Milk Chocolate Digestive,,,,
-resources/images/MCDTop.jpg,,,100%,image-local,1,Top view of a Milk Chocolate Digestive,Milk Chocolate Digestive,,,,
-resources/images/MCDSide.jpg,,,100%,image-local,3,Side view of a Milk Chocolate Digestive   ,Milk Chocolate Digestive,,,,
-,,,,,,,,,,,
-resources/images/ShortcakeTop.jpg,,,100%,image-local,,Top view of a Shortcake,Shortcake,,,,
-resources/images/ShortcakeOblique.jpg,,,100%,image-local,,Oblique view of a Shortcake,Shortcake,,,,
-resources/images/ShortcakeOblique.jpg,,,100%,image-local,,Köhler’s Medizinal-Pflanzen,Shortcake,,,,
+URI,SmallURI,LargeURI,ImageWidth,Type,Priority,Caption,Taxon,Character,State,UseFor,TipStyle
+resources/text/demo.html,,,,Html-local,1,,"William Raybould,…",,,,

```

Comment:
Is the "William Rayboud,..." part correct? (Might be intended to be "William Rayboud",...,)

Reply:
The "William Rayboud,..." is correct - the title truncation is very "by hand" at this point, so this is the title as it shows up in the UI. 

The line endings, however, is more interesting, and I'll fix those. I blame Excel.

Reply:
Line endings should now be resolved?